### ISSUE TITLE
log error if a fiber terminates due to uncaught Lua error

### ISSUE THREAD

**Issue by mejedi** [tarantool/tarantool#1238]
If an error occurs in a fiber spawned by Lua it silently ceases to exist. It would be nice to log the error.

**Comment by rtsisyk** [comment]
It is a regression since 1.6.7 caused by src/fiber.c(cc) refactoring (fiber_loop() function):

```
./src/tarantool: version 1.6.7-562-g3dbc9ff
type 'help' for interactive help
tarantool> fiber = require('fiber')
---
...

tarantool> fiber.create(function() fiber.sleep(2); error('XXX') end)
---
- status: suspended
  name: lua
  id: 102
...

tarantool> ./src/tarantool: LuajitError: [string "return fiber.create(function() fiber.sleep(2)..."]:1: XXX

---
...
```

```
./src/tarantool: version 1.6.8-282-gd629c07
type 'help' for interactive help
tarantool> fiber = require('fiber')
---
...

tarantool> fiber.create(function() fiber.sleep(2); error('XXX') end)
---
- status: suspended
  name: lua
  id: 102
...

tarantool> fiber.info()
---
- 101:
    csw: 6
    backtrace: []
    fid: 101
    memory:
      total: 65776
      used: 0
    name: interactive
...

```

**Comment by kostja** [comment]
* Roman Tsisyk <notifications@github.com> [15/12/30 23:33]:

OK, but don't log FiberIsCancelled please, I tried to remove
logging of this error, and apparently killed all logging.


-- 
Konstantin Osipov, Moscow, Russia, +7 903 626 22 32
http://tarantool.org - www.twitter.com/kostja_osipov

**Comment by rtsisyk** [comment]
Fixed in 1.6.8-371-g6a7d814
